redbot/cogs/modlog/modlog.py

```diff
@@ -117,7 +117,13 @@ async def case(self, ctx: commands.Context, number: int):
             if await ctx.embed_requested():
                 await ctx.send(embed=await case.message_content(embed=True))
             else:
-                await ctx.send(await case.message_content(embed=False))
+                message = _("{case}\n**Timestamp**: {timestamp}").format(
+                    case=await case.message_content(embed=False),
+                    timestamp=datetime.fromtimestamp(case.created_at).strftime(
+                        "%Y-%m-%d %H:%M:%S"
```

Comment:
Updated handling per #4017:
```suggestion
                    timestamp=datetime.utcfromtimestamp(case.created_at).strftime(
                        "%Y-%m-%d %H:%M:%S UTC"
```